networking/understanding-networking.adoc

```diff
@@ -12,4 +12,13 @@ IP address means that Pods can be treated like physical hosts or virtual
 machines in terms of port allocation, networking, naming, service discovery,
 load balancing, application configuration, and migration.
 
+[NOTE]
+====
+Link-local IP addresses are assigned from the IPv4 `169.254.0.0/16` CIDR block. Some cloud platforms provide APIs reachable from only the link-local CIDR block.
```

Comment:
The first sentence makes it sound like someone is assigning IPs from that block to pods (or nodes), but that's not accurate. OCP itself does not make any use of link-local IPs. And the problem isn't that the cloud only allows access _from_ link-local IPs, it's that the cloud API is provided _on_ a link-local IP, and OCP doesn't allow access _to_ that IP from the pod network.

The top-level story here is "OCP blocks access to cloud metadata APIs, but you can use hostNetwork to get around that". The fact that the IPv4 link-local network block is involved is not really that important beyond the fact that if someone searches the docs for "IPv4 link local" we want them to find this note. Also, probably, if they search for "metadata api" or "169.254.169.254" because those are the other things they might search for when they're trying to figure out why their pod can't access the API it's trying to access.